rsrcdump/jsonio.py

```diff
@@ -94,12 +95,18 @@ def resource_fork_to_json(
                 else:
                     sanitized_name = ""
                 if sanitized_name:
-                    if isinstance(res_id, bytes):
-                        res_id = res_id.decode("utf-8")
+                    # Subclass of "int" and "bytes" cannot exist: would
+                    # have incompatible method signatures
+                    # Statement is unreachable
+                    ##if isinstance(res_id, bytes):
+                    ##    res_id = res_id.decode("utf-8")
                     filename = F"{res_id}.{sanitized_name}{ext}"
                 else:
-                    if isinstance(res_id, bytes):
-                        res_id = res_id.decode("utf-8")
+                    # Subclass of "int" and "bytes" cannot exist: would
+                    # have incompatible method signatures
+                    # Statement is unreachable
+                    ##if isinstance(res_id, bytes):
+                    ##    res_id = res_id.decode("utf-8")
```

Comment:
We can remove all +/- lines here (res_id should never be bytes)

Reply:
Removed in 0e63227